src/main/java/uz/pdp/spring_boot_security_web/service/SubjectService.java

```diff
@@ -50,12 +50,34 @@ public SubjectEntity add(SubjectRequestDTO subjectRequestDTO) {
         return subjectRepository.save(subjectEntity);
     }
 
-
     public SubjectEntity getByTitle(String title) {
         Optional<SubjectEntity> byId = subjectRepository.findByTitle(title);
         if (!byId.isPresent()) {
-            throw new IllegalArgumentException(title + " Subject not found");
```

Comment:
Yoq subjectni titleni olmoqchi bolgansiz bu aniq hato

Reply:
HA tushundim hozir to'g'irlayman